backends/qualcomm/qnn_quantizer.py

```diff
@@ -124,7 +126,7 @@ def get_default_qnn_ptq_config(
         quant_min=0,
         quant_max=255,
         qscheme=torch.per_tensor_affine,
-        observer_or_fake_quant_ctr=HistogramObserver.with_args(**extra_args),
+        observer_or_fake_quant_ctr=MovingAverageMinMaxObserver.with_args(**extra_args),
```

Comment:
I am not sure if this is a good idea as a blanket change for of qnn quantizer. Why do you need this change?

Reply:
Thanks for pointing it out. There seemed to be an error before when using HistogramObserver for float.min.
But it seems to work now.

Reply:
Yes, there is an issue with histogram observer for float.min that @jerryzh168 is aware of.

Do you still need this change? If so, I would suggest using this only where needed, otherwise you are using min/max observer for everything

Reply:
I have changed it back in new commit. Thanks.